LICENSE.rst

```diff
@@ -4,14 +4,53 @@
 Licenses
 ########
 
-DFHack is distributed under a range of permissive and weakly copyleft licenses.
-
-The core uses the ZLib license; the others are described below.
-
-License of DFHack
-=================
-https://github.com/peterix/dfhack
-Copyright (c) 2009-2012 Petr Mrázek (peterix@gmail.com)
+DFHack is distributed under the Zlib license, with some MIT-
+and BSD-licensed components.  These licenses protect your right
+to use DFhack for any purpose, distribute copies, and so on.
+
+The core, plugins, scripts, and other DFHack code all use the
+ZLib license unless noted otherwise.  By contributing to DFHack,
+authors release the contributed work under this license.
+
+DFHack also draws on several external packages.
+Their licenses are summarised here and reproduced below.
+
+=============== ============= =================================================
+Component       License       Copyright
+=============== ============= =================================================
+DFHack_         Zlib          (c) 2009-2012, Petr Mrázek
```

Comment:
I'll escape them, given that it works that way in Sphinx too - the licenses are one of the more important areas to format correctly!